azure-pipelines.yml

```diff
@@ -57,6 +57,16 @@ stages:
 
         sudo python3 -m pip install dist/asyncsnmp-2.1.0-py3-none-any.whl
         python3 setup.py test
+        
+        # Install .NET CORE
+        wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+        sudo dpkg -i packages-microsoft-prod.deb
+        sudo apt-get update
+        sudo apt --fix-broken install -y
+        #sudo apt-get install libhiredis0.14 libnl-3-200 libnl-route-3-200
+        sudo apt-get install -y apt-transport-https
+        sudo apt-get update
+        sudo apt-get install -y dotnet-sdk-5.0
       displayName: "Unit tests"
```

Comment:
It is better to create a new step to install the .net core, not in the "Unit tests" step.